main.cpp

```diff
@@ -164,7 +167,7 @@ int main(int argc, char **argv) {
             return FunctionAction::MakePrivate;
           } else if (function.getName().equals("b")) {
             return FunctionAction::MakePublic;
-          } else if (function.getName().equals("c")) {
+          } else if (function.getName().equals("cc")) {
             return FunctionAction::MakePublic;
           } else {
             return FunctionAction::Skip;
```

Comment:
It is just that I had renamed the function in the c file and it had to be renamed here too.

Reply:
Yes, but why is `cc` special cased (or "why is `c` special case) or whatever? I don't understand why we have any function names that are special cased.

Reply:
Oh because when moved and compiled `c` was becoming `libc.so`.